Updating kryo-serializers to 0.45 fails to build

```
$ mvn clean install
...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  9.978 s
[INFO] Finished at: 2021-01-28T12:16:25-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project disq: Compilation failure: Compilation failure:
[ERROR] /working/disq/src/main/java/org/disq_bio/disq/serializer/DisqKryoRegistrator.java:[177,51] <anonymous org.disq_bio.disq.serializer.DisqKryoRegistrator$1> is not abstract and does not override abstract method read(com.esotericsoftware.kryo.Kryo,com.esotericsoftware.kryo.io.Input,java.lang.Class<? extends org.disq_bio.disq.impl.file.HadoopFileSystemWrapper>) in com.esotericsoftware.kryo.Serializer
[ERROR] /working/disq/src/main/java/org/disq_bio/disq/serializer/DisqKryoRegistrator.java:[187,42] name clash: read(com.esotericsoftware.kryo.Kryo,com.esotericsoftware.kryo.io.Input,java.lang.Class<org.disq_bio.disq.impl.file.HadoopFileSystemWrapper>) in <anonymous org.disq_bio.disq.serializer.DisqKryoRegistrator$1> and read(com.esotericsoftware.kryo.Kryo,com.esotericsoftware.kryo.io.Input,java.lang.Class<? extends org.disq_bio.disq.impl.file.HadoopFileSystemWrapper>) in com.esotericsoftware.kryo.Serializer have the same erasure, yet neither overrides the other
[ERROR] /working/disq/src/main/java/org/disq_bio/disq/serializer/DisqKryoRegistrator.java:[186,11] method does not override or implement a method from a supertype
[ERROR] /working/disq/src/main/java/org/disq_bio/disq/serializer/DisqKryoRegistrator.java:[195,48] <anonymous org.disq_bio.disq.serializer.DisqKryoRegistrator$2> is not abstract and does not override abstract method read(com.esotericsoftware.kryo.Kryo,com.esotericsoftware.kryo.io.Input,java.lang.Class<? extends org.disq_bio.disq.impl.file.NioFileSystemWrapper>) in com.esotericsoftware.kryo.Serializer
[ERROR] /working/disq/src/main/java/org/disq_bio/disq/serializer/DisqKryoRegistrator.java:[203,39] name clash: read(com.esotericsoftware.kryo.Kryo,com.esotericsoftware.kryo.io.Input,java.lang.Class<org.disq_bio.disq.impl.file.NioFileSystemWrapper>) in <anonymous org.disq_bio.disq.serializer.DisqKryoRegistrator$2> and read(com.esotericsoftware.kryo.Kryo,com.esotericsoftware.kryo.io.Input,java.lang.Class<? extends org.disq_bio.disq.impl.file.NioFileSystemWrapper>) in com.esotericsoftware.kryo.Serializer have the same erasure, yet neither overrides the other
[ERROR] /working/disq/src/main/java/org/disq_bio/disq/serializer/DisqKryoRegistrator.java:[202,11] method does not override or implement a method from a supertype
[ERROR] -> [Help 1]
```

--- a/pom.xml
+++ b/pom.xml
@@ -41,7 +41,7 @@
         <htsjdk.version>2.24.1</htsjdk.version>
         <hadoop.version>3.2.1</hadoop.version>
         <mockito.version>3.7.7</mockito.version>
-        <kryo-serializers.version>0.43</kryo-serializers.version>
+        <kryo-serializers.version>0.45</kryo-serializers.version>
         <slf4j.version>1.7.30</slf4j.version>
         <scala.version>2.12</scala.version>
         <spark.version>3.1.2</spark.version>
@@ -57,6 +57,12 @@
             <groupId>de.javakaffee</groupId>
             <artifactId>kryo-serializers</artifactId>
             <version>${kryo-serializers.version}</version>
+            <exclusions>
+              <exclusion>
+                <groupId> com.esotericsoftware</groupId>
+                <artifactId>kryo</artifactId>
+              </exclusion>
+            </exclusions>
         </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>

